### PR TITLE
Fixing the security-check notification system

### PR DESCRIPTION
**Pull request by transcedentalia**

### CODE REVIEW

**Comment by bobtfish** [comment]
👍

File: paasta_tools/monitoring_tools.py [hunk 1]
```diff
@@ -169,32 +170,44 @@ def send_event(service, check_name, overrides, status, output, soa_dir, ttl=None
     :param output: The output to emit for this event
     :param soa_dir: The service directory to read monitoring information from
     """
+    system_paasta_config = load_system_paasta_config()
+    sensu_host = system_paasta_config.get_sensu_host()
```
Comment (Rob-Johnson) [inline paasta_tools/monitoring_tools.py]: doesn't look like these are used anywhere?

File: paasta_tools/monitoring_tools.py [hunk 2]
```diff
@@ -169,32 +170,44 @@ def send_event(service, check_name, overrides, status, output, soa_dir, ttl=None
...
+    sensu_port = system_paasta_config.get_sensu_port()
+
+    if not sensu_host:
+        return
```
Comment (Rob-Johnson) [inline paasta_tools/monitoring_tools.py]: what's wrong with the default?

**Comment by solarkennedy** [comment]
Instead of making a new function (`send_event` AND `send_event_to_sensu`) can you adjust send_event to meet your needs?

Even better, just default to 'localhost' ?

**Pull request merged**